manim/utils/tex_file_writing.py

```diff
@@ -22,19 +22,49 @@ def tex_hash(expression):
 
 
 def tex_to_svg_file(expression, environment=None, tex_template=None):
-    """Takes a tex expression and returns the svg version of the compiled tex"""
+    """Takes a tex expression and returns the svg version of the compiled tex
+
+    Parameters
+    ----------
+    expression : :class:`str`
+        String containing the TeX expression to be rendered, e.g. ``\\sqrt{2}`` or ``foo``
+    environment : :class:`str`
+        The string containing the environment in which the expression should be typeset, e.g. ``align*``
+        Optional. If not set, the expression will be rendered in text mode.
+    tex_template : :class:`TexTemplate`
+        Template class used to typesetting.
+        Optional. If not set, the default template will be used.
```

Comment:
Technically, it will use whatever template is set in config["tex_template"] iirc.
This is the default template unless a different template was set in the config.
That could happen if:
1. You manually change config["tex_template"]
2. You specify a --tex_template file on CLI
3. You specify a tex template in the .cfg file.

Reply:
but that's ok.
We can always claim that "the default" is the globally configured template and not any particular template....